Windows/MainWindow.cpp

```diff
@@ -281,6 +283,10 @@ namespace MainWindow
 			NativeMessageReceived("gpu resized", "");
 		}
 
+		if (screenManager) {
+			screenManager->RecreateAllViews();
```

Comment:
Hmm.  Maybe we should recreate views when the resized event comes through to the DisplayLayoutEditor?  I mean via `void DisplayLayoutScreen::sendMessage(const char *message, const char *value) {`.

-[Unknown]